src/utils/user_utils.js

```diff
@@ -21,21 +21,21 @@ export function getFullName(user: UserProfile): string {
 export function displayUsername(
     user: UserProfile,
     teammateNameDisplay: string,
-    usernameWithPrefix: boolean,
+    useDefaultUserName: boolean = true,
 ): string {
-    let name = localizeMessage('channel_loader.someone', 'Someone');
+    let name = useDefaultUserName ? localizeMessage('channel_loader.someone', 'Someone') : '';
 
     if (user) {
         if (teammateNameDisplay === Preferences.DISPLAY_PREFER_NICKNAME) {
             name = user.nickname || getFullName(user);
         } else if (teammateNameDisplay === Preferences.DISPLAY_PREFER_FULL_NAME) {
             name = getFullName(user);
         } else {
-            name = usernameWithPrefix ? `@${user.username}` : user.username;
+            name = user.username;
         }
 
         if (!name || name.trim().length === 0) {
-            name = usernameWithPrefix ? `@${user.username}` : user.username;
+            name = user.username;
```

Comment:
what in the entire product is affected by this change?

Reply:
There is no function that uses this param in the codebase anymore.

I could have used another param but using two flag params as arguments didn't feel right at all because I would have to change every place this function is used(That is a lot!) and pass the third params value even if it is default value to keep the fourth param optional.

The other option is to use another function.

Reply:
cool, just wanted to make sure, if that is the case we can leave the change